rclcpp/include/rclcpp/experimental/timers_manager.hpp

```diff
@@ -271,6 +272,15 @@ class TimersManager
       return weak_heap_.empty();
     }
 
+    /**
+     * @brief Returns the size of the heap.
+     * @return the number of valid timers in the heap.
+     */
+    size_t size() const
+    {
+      return weak_heap_.size();
+    }
+
```

Comment:
this method is not used currently, so should be removed from this PR?

Reply:
Good call, removed